frame/nomination-pools/src/lib.rs

```diff
@@ -2098,6 +2173,23 @@ pub mod pallet {
 			ensure!(bonded_pool.can_nominate(&who), Error::<T>::NotNominator);
 			T::Staking::chill(&bonded_pool.bonded_account())
 		}
+
+		/// Pool Member set reward claim permission
```

Comment:
I don't mean to be too strict on language as I am not a native english speaker either, but is a bit too unstructured :)

Reply:
what would you suggest here?

I need to put more effort on universal documentation, I'll get right on it